examples/src/bin/gpio_interrupt.rs

```diff
@@ -61,9 +62,9 @@ fn main() -> ! {
     }
 }
 
+#[handler]
 #[ram]
```

Comment:
Should `#[handler]` imply `#[ram]`?

Reply:
I guess while most users will use `#[ram]` I think having it explicit looks a bit "cleaner" - no real reasoning ... it just feels like that for me but don't have a hard opinion on that

Reply:
My only worry is if they forget to add this they could incur some interrupt latency if the handler is outside of the cache. I think if we use it everywhere in examples, we can hope that most will also add it in their own code.

Reply:
I wonder if we could (someday) have a debug feature to check interrupt handling code doesn't trigger a cache miss (at runtime). I think doing that via static analysis (with some additional tooling) is possible but a lot of effort 🤔